tensorly/tenalg/proximal.py

```diff
@@ -879,11 +879,13 @@ def hals_nnls(
     UtU,
     V=None,
     n_iter_max=500,
-    tol=10e-8,
+    tol=1e-8,
     sparsity_coefficient=None,
-    normalize=False,
+    ridge_coefficient=None,
     nonzero_rows=False,
     exact=False,
+    epsilon=0,
```

Comment:
```suggestion
    epsilon=0.0,
```

Many tools will infer this to be an int if 0 is the default.